### ISSUE TITLE
Typo in string parameter for balloon description

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#356]
- [x] Typo in string parameter for balloon descriptions under balloon headings "{{zero}} more negative charges than positive charges."

This descriptions should be:
"{{no}} more negative charges than positive charges."

All other parameters for the balloon description seem fine.

**Comment by jessegreenberg** [comment]
Done in the above commit, closing.
